From 271315d8b35c4ae72666444ec2c6338ca9604994 Mon Sep 17 00:00:00 2001
From: Petr Mladek <pmladek@walk.suse.cz>
Date: Wed, 18 Aug 2010 15:47:14 +0000
Subject: Non-translated Tools/Options/OpenOffice.org Writer/Comparison (bnc#615000)

* src/sdf/writer-comparison-option-tree.sdf: the translations must be shifted
  in the itemlist (bnc#615000)
---
diff --git a/src/sdf/writer-comparison-option-tree.sdf b/src/sdf/writer-comparison-option-tree.sdf
new file mode 100644
index 0000000..3a483c1
--- dev/null
+++ openoffice.org-3.3.0/ooo-build-28d2a4e7af25256f2919b81d6fe03426ab2c7454/src/sdf/writer-comparison-option-tree.sdf
@@ -0,0 +1,347 @@
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	af	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ar	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ast	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	bg	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	bn	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	br	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	brx	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	bs	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ca	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	cs	Srovnání				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	cy	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	da	Sammenligning				20090726 13:15:11
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	de	Vergleich				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	dgo	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	el	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	en-GB	Comparison				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	en-ZA	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	eo	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	es	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	es	Comparación				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	et	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	eu	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	fa	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	fi	Vertailu				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	fr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ga	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	gl	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	gu	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	hi	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	hr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	hu	Összehasonlítás				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	id	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	is	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	it	Confronto				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ja	比較				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	kk	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	km	ការ​ប្រៀបធៀប				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	kn	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	kok	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ko	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ks	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	lv	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	mai	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	mk	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ml	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	mni	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	mn	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	mr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	my	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	nb	Sammenligning				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ne	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	nl	Vergelijken				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	nn	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	nr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ns	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	oc	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	om	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	or	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	pl	Porównanie				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	pt-BR	Comparação				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	pt	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ro	Comparaţie				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ru	Сравнение				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	rw	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sa-IN	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sat	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sd	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sh	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	si	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sk	Porovnanie				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sl	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ss	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	st	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sv	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	sw-TZ	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ta	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	te	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	tg	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	tn	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	tr	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ts	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	ve	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	vi	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	xh	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	zh-CN	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	12			0	zh-TW	Comparison				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	af	Versoenbaarheid				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ar	التوافق				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ast	Compatibilidá				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	bg	Съвместимост				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	bn	সুসংগতি				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	br	Keverlec'hded				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	brx	गोरोबनाय गोनां				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	bs	Kompatibilnost				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ca	Compatibilitat				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	cs	Kompatibilita				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	cy	Cytunedd				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	da	Kompatibilitet				20090726 13:15:11
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	de	Kompatibilität				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	dgo	अनुकूलता 				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	el	Συμβατότητα				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	en-GB	Compatibility				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	en-ZA	Compatibility				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	eo	Kongrueco				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	es	Compatibilidad				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	es	Compatibilidad				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	et	Ühilduvus				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	eu	Bateragarritasuna				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	fa	سازگاری				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	fi	Yhteensopivuus				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	fr	Compatibilité				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ga	Comhoiriúnacht				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	gl	Compatibilidade				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	gu	સુસંગતતા				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	hi	सुसंगतता				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	hr	Kompatibilnost				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	hu	Kompatibilitás				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	id	Kompatibilitas				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	is	Samhæfni				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	it	Compatibilità				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ja	互換性				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	kk	Сәйкестiру				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	km	ភាព​ឆប​គ្នា				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	kn	ಸಹವರ್ತಿನೀಯತೆ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	kok	सुसंगतताय				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ko	호환성				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ks	مناسبت				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	lv	Savietojamība				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	mai	सुसंगतता				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	mk	Компатибилност				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ml	പൊരുത്തം				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	mni	চুন্নবা				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	mn	Нийцтэй байдал				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	mr	सहत्वता				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	my	လိုက်ဖက်ညီမှု				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	nb	Samspill				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ne	मिल्दोपना				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	nl	Compatibiliteit				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	nn	Samspel				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	nr	Ukuhlala ngokuhlangana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ns	Tumelelano				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	oc	Compatibilitat				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	om	Walta'iinsa				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	or	ସୁସଂଗତି				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	pl	Zgodność				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	pt-BR	Compatibilidade				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	pt	Compatibilidade				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ro	Compatibilitate				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ru	Совместимость				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	rw	Bihuye neza				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sa-IN	योग्यता				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sat	सुसंगतता				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sd	موافقت				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sh	Saglasnost				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	si	CJK යෝග්‍යතාව				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sk	Kompatibilita				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sl	Združljivost				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sr	Сагласност				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ss	Kusebentisana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	st	Tumellano				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sv	Kompatibilitet				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	sw-TZ	Utangamano				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ta	ஒத்தியல்பு				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	te	అనుసంధానం మాత్రమే				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	tg	Ҳамоҳангӣ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	tn	Tshwantshanyo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	tr	Uyumluluk				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ts	Ku tirhisana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	ve	U elana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	vi	Sự tương thích				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	xh	Ungqinelwano				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	zh-CN	兼容性				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	zh-TW	相容性				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	13			0	zu	Ukuhambelana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	af	OutoByskrif				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ar	تسمية توضيحيّة تلقائيّة				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	bg	Автонадпис				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	bn	স্বয়ংক্রিয়-ক্যাপশন				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	br	Leadell emgefreek				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	brx	गावआरि केप्शन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	bs	Auto-naslov				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ca	Llegenda automàtica				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	cs	Automatický popisek				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	cy	AwtoBennawd				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	da	Autobilledtekst				20090726 14:15:11
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	de	AutoBeschriftung				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	dgo	स्वचलत सिरलेख				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	el	Αυτόματη λεζάντα				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	en-GB	AutoCaption				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	en-ZA	AutoCaption				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	eo	Aŭtomata apudskribo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	es	Autotítulo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	es	Autotítulo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	et	Automaatpealdis				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	eu	Epigrafe automatikoa				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	fa	عنوان خودکار				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	fi	Automaattiotsikointi				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	fr	Légende automatique				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ga	Foscríbhinn Uathoibríoch				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	gl	Lenda automática				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	gu	સ્વયંકૅપ્શન				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	hi	स्वचालितशीर्षक				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	hr	SamoOpis				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	hu	Automatikus felirat				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	id	KapsiOtomatis				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	is	Sjálfvirkar fyrirsagnir (AutoCaption)				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	it	Didascalia automatica				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ja	自動図表番号				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	kk	Автотақырыбы				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	km	ចំណងជើង​ស្វ័យប្រវត្តិ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	kn	ಸ್ವಯಂಶೀರ್ಷಿಕೆ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	kok	आपसूक कॅप्शन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ko	자동 캡션				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ks	پانئی كیپشن				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	lv	Autom. paraksts				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	mai	स्वचालितकैप्शन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	mk	Автоматски наслов				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ml	ഓട്ടോകാപ്ഷന്‍				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	mni	ওতোকেপসন				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	mn	Автомат бичээс				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	mr	स्वयंकॅप्शन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	my	အလိုအလျောက်စာတန်း				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	nb	Automatisk bildetekst				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ne	स्वत: क्याप्सन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	nl	Automatische bijschriften				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	nn	Automatisk bilettekst				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	nr	Ikhephtjhini yokuzenzakalela				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ns	Go dira Ditlhakakgolo mo go Itiragalelago				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	om	OfinGodayyaa				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	or	ସ୍ବଯଂଚାଳିତଶିରୋନାମା				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	pl	Autopodpis				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	pt-BR	Legenda automática				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	pt	Legenda automática				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ro	Titlu automat				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ru	Автозаголовок				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	rw	NyaMwiranga				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sa-IN	स्वचालितः अधोलेखः				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sat	स्वचालितकैप्शन				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sd	خودڪار سرو				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sh	Automatsko naslovljavanje				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	si	ස්වයංක්‍රීය මාතෘකා				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sk	Automatický popisok				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sl	Samopoimenovanje				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sq	AutoCaption...				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sr	Аутоматско насловљавање				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ss	Kutibhalekela				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	st	KhepsheneeIketsahallang				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sv	Autobeskrivning				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	sw-TZ	AutoCaption				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ta	தானானதலைப்பு				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	te	 స్వయంక్లుప్తవివరణ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	tg	Автодохилкунӣ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	tr	Otomatik başlık				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ts	Nhlokomhaka yo titela hi yoxe				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	ve	OthoKhephusheni				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	vi	Cchú dẫn tự động				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	xh	Inkcazo NgemifanekisoEzenzekelayo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	zh-CN	自动插入标题				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	zh-TW	自動標籤				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	14			0	zu	Isihloko				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	af	Massapos-e-pos				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ar	رسالة إلكترونيّة عبر دمج المراسلات				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	bg	Циркулярни писма по е-поща				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	bn	মেইল একত্রীকরণ ইমেইল				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	br	Postel mailing				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	brx	मेल मार्ज इ-मेइल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	bs	Cirkularno pismo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ca	Adreça electrònica de la combinació de correu				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	cs	Hromadná korespondence elektronickou poštou				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	cy	E-bost Cyfuno E				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	da	Brevfletning e-mail				20090726 14:15:11
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	de	Serienbrief-E-Mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	dgo	  ई-मेल मेल-विलय करो				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	el	Συγχώνευση ηλεκτρονικής αλληλογραφίας				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	en-GB	Mail Merge E-mail				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	en-ZA	Mail Merge E-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	eo	Retpoŝta kunfandado				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	es	Combinar correspondencia				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	es	Combinar correspondencia				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	et	E-posti sätted kirjakoostel				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	eu	Posta-fusioko mezua				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	fa	پست الکترونیکی Mail Merge				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	fi	Joukkokirje-sähköposti				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	fr	E-mail de mailing				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ga	Postchumasc				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	gl	Combinación de correspondencia para envío electrónico				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	gu	મેઇલ એકત્ર ઇમેઇલ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	hi	मेल मर्ज ई-मेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	hr	Pošta Sjedini elektroničku poštu				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	hu	Körlevél e-mailben				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	id	Surel Surat Massal				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	is	Netfang fyrir bréfblöndun				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	it	Stampa in serie e-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ja	差し込み印刷 E-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	kk	Стандарттық хаттарды жасауға арналған  е-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	km	សំបុត្រ​សំណុំ​បែប​បទ				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	kn	ಮೈಲ್ ವಿಲೀನದ ಇ-ಮೈಲ್				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	kok	मेल मर्ज ई-मेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ko	편지 병합 전자 메일				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ks	میل رلئویو  E-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	lv	E-pasta sapludināšana				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	mai	डाक मिलाबू ईमेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	mk	Циркуларна е-пошта				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ml	മെയില്‍ മേര്‍ജ് ഈമെയില്‍				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	mni	মেইল পুনশিনবা ই-মেইল				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	mn	Цуврал захиа				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	mr	मेल एकत्र ई-मेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	my	အီးမေးလ်စာပို့စနစ်				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	nb	Brevfletting av e-post				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ne	पत्र गाभ्ने इमेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	nl	Standaardbrief E-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	nn	Brevfletting av e-post				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	nr	Ukuhlangana kweposo ye-E-meyili				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ns	Kopanya Poso ya E-meile				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	om	Ergaa-e Makiinsa Ergannoo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	or	ଇ-ମେଲ ମେଲ ମିଶାଅ 				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	pl	Korespondencja seryjna e-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	pt-BR	Mala direta de e-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	pt	Correio electrónico de impressão em série				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ro	Trimite email în serie				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ru	Рассылка писем				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	rw	Imeyili Mukomatanyabutumwa				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sa-IN	सन्देशनिमज्जन ई-मेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sat	डाक मिलायें ईमेल				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sd	جزب ڪيل اي۔ميل 				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sh	Cirkularna e-pošta				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	si	මේල් මර්ජ් විද්‍යුත් ලිපි				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sk	Hromadná korešpondencia emailom				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sl	Spajanje e-pošte				20100415 16:29:55
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sq	Bashkangjitje me të dhëna				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sr	Циркуларна е-пошта				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ss	I-imeyili yekuhlanganisa liposi				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	st	I-meili ya Kopanyo ya Poso				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sv	Kopplad utskrift med e-post				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	sw-TZ	Mail Merge E-mail				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ta	அஞ்சல் அமிழ் மின்னஞ்சல்				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	te	తపాళ కలయిక ఇ తపాలా				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	tg	Номаи Mail Merge				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	tr	Posta Birleştirme Adresi				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ts	E-meyili ya Mail Merge				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	ve	Emeili ya Thanganyo ya Manwalo				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	vi	Trộn thư điện tử				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	xh	Imeyile Yomanyano Lweposi				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	zh-CN	邮件合并电子邮件				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	zh-TW	合併列印電子郵件				2002-02-02 02:02:02
+svx	source\cui\treeopt.src	0	itemlist	RID_OFADLG_OPTIONS_TREE_PAGES.SID_SW_EDITOPTIONS	15			0	zu	Ukuhlanganiswa Ne-imayili				2002-02-02 02:02:02
--
cgit v0.8.3-6-g21f6
